src/ansible_navigator/configuration_subsystem/navigator_post_processor.py

```diff
@@ -281,6 +281,20 @@ def execution_environment(self, entry: SettingsEntry, config) -> PostProcessorRe
                         )
                     if hint:
                         exit_messages.append(ExitMessage(message=hint, prefix=ExitPrefix.HINT))
+
+            # check for /dev/mqueue when using podman because runner passes ipc=host
+            # https://github.com/ansible/ansible-navigator/issues/610
+            if config.container_engine == "podman" and not Path("/dev/mqueue").exists():
```

Comment:
Maybe use is_dir since the error message claims exactly that?

Reply:
added it, good idea